assets/dev.go

```diff
@@ -11,9 +11,12 @@ import (
 )
 
 // Assets contains files that will be included in the binary
+// this is a union file system, so to reach the expected file
+// the root folder defined in the map should be prepended
+// to the file path
 var Assets = union.New(map[string]http.FileSystem{
 	"/migrations": http.Dir(importPathToDir("github.com/k-box/k-link-registry/assets/migrations")),
-	"/static":     http.Dir(importPathToDir("github.com/k-box/k-link-registry/assets/static")),
+	"/static":     http.Dir(importPathToDir("github.com/k-box/k-link-registry/ui/dist")),
```

Comment:
Probably this is an improvement. The only downside is: now the project (backend only) will no longer start without the frontend present. I'm okay with that, since the project gained the capability to be compiled under windows.

Reply:
Once I'll get confident, I'll make it working again without pre-compiled ui files